envi/archs/arm/disasm.py

```diff
@@ -1823,7 +1823,7 @@ def getOperValue(self, op, emu=None):
         addr = self.getOperAddr(op, emu)
         return emu.readMemValue(addr, self.tsize)
 
-    def setOperValue(self, op, emu=None, val=None):
+    def setOperValue(self, op, emu=None, val=None):     # Duplicate or typo? setOperAddr ???
```

Comment:
good catch!  delete one of them.  it looks like they are identical.  if you see that they are not, please let me know.  currently, the second one is being used, because it's defined last.